force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -94,7 +129,16 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the input is "Empty", "Null", or "Contains Text".
      */
     public static String checkString(String a) {
-        return null; // Replace null with the variable you used to store the result
+        String textOrNot;
+
+        if (a == '') {
+            textOrNot = 'Empty';
+        } else if (a == null) {
+            textOrNot = 'Null';
+        } else {
+            textOrNot = 'Contains Text';
+        }
+        return textOrNot; // Replace null with the variable you used to store the result
```

Comment:
This is solution is totally fine. Sometimes when I am working I like to leave what I consider the main code path of the method outside the conditional. I think it makes things a litte more clear. But nothing is wrong with this. Here is my solution.

 public static String checkString(String a) {
        if (a == null) {
            return 'Null';
        } else if (String.isEmpty(a)) {
            return 'Empty';
        }
        return 'Contains Text';
    }